src/lib/eventStack/EventTarget.js

```diff
@@ -81,7 +81,10 @@ export default class EventTarget {
     const handler = this.createEmitter(eventType, this.pools)
 
     this.handlers.set(eventType, handler)
-    this.target.addEventListener(eventType, handler)
+    this.target.addEventListener(eventType, handler, {
+      capture: true,
+      passive: true,
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

We need `capture` to fix the issue.
We can use `passive` because we don't use `preventDefault()` on events.